test/Microsoft.AspNetCore.Antiforgery.Test/DefaultAntiforgeryTest.cs

```
@@ -393,11 +386,13 @@ public void ValidateTokens_FromInvalidStrings_Throws()
                 .Setup(o => o.Deserialize("form-token"))
                 .Returns(context.TestTokenSet.RequestToken);
 
+            string message;
             context.TokenGenerator
-                .Setup(o => o.ValidateTokens(
+                .Setup(o => o.TryValidateTokenSet(
                     context.HttpContext,
                     context.TestTokenSet.OldCookieToken,
-                    context.TestTokenSet.RequestToken))
+                    context.TestTokenSet.RequestToken,
+                    out message))
                 .Throws(new InvalidOperationException("my-message"));
```

Why is `TryValidateTokenSet` set up to `throw` instead of returning `false`?